yapapi/executor/subtask.py

```diff
@@ -82,58 +82,58 @@ def _stop(self, retry: bool = False):
 
     @staticmethod
     def for_handle(
-        handle: Handle["Task[TaskData, TaskResult]"],
-        queue: SmartQueue["Task[TaskData, TaskResult]"],
+        handle: Handle["SubTask[Data, Result]"],
+        queue: SmartQueue["SubTask[Data, Result]"],
         emitter: Callable[[events.Event], None],
-    ) -> "Task[TaskData, TaskResult]":
-        task = handle.data
-        task._handle = (handle, queue)
-        task._start(emitter)
-        return task
+    ) -> "SubTask[Data, Result]":
+        subtask = handle.data
+        subtask._handle = (handle, queue)
+        subtask._start(emitter)
+        return subtask
 
     @property
-    def data(self) -> TaskData:
+    def data(self) -> Data:
         return self._data
 
     @property
-    def output(self) -> Optional[TaskResult]:
+    def output(self) -> Optional[Result]:
         return self._result
 
     @property
     def expires(self) -> Optional[datetime]:
         return self._expires
 
-    def accept_task(self, result: Optional[TaskResult] = None) -> None:
-        """Accept task that was completed.
+    def accept_result(self, result: Optional[Result] = None) -> None:
+        """Accept subtask that was completed.
```

Comment:
and _here_ it should rather be a "job" -> since we're accepting a _result_ of a single instance of a fragment...

Reply:
or just "result" ...